src/swarm-engine/Swarm/Util/Content.hs

```diff
@@ -0,0 +1,56 @@
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+module Swarm.Util.Content where
```

Comment:
Can we add a short comment explaining what kinds of things can be found in this module?